src/feature_annotation/score_genes_cell_cycle_scanpy/config.vsh.yaml

```diff
@@ -0,0 +1,81 @@
+functionality:
+  name: score_genes_cell_cycle_scanpy
+  namespace: feature_annotation
+  description: |
+    Calculates the score associated to S phase and G2M phase and annotates the cell cycle phase for each cell, as implemented by scanpy. 
+    The score is the average expression of a set of genes subtracted with the average expression of a reference set of genes.
+  argument_groups:
+
+    - name: Inputs
+      arguments:
+        - name: --input
+          type: file
+          description: Input h5mu file
+          required: true
+          example: input_file.h5mu
+        - name: --s_genes
+          type: file
+          description: |
+            Path to a .txt file containing the gene list of s phase genes to be scored. 
+            The gene list file should be formatted as a single column with gene symbols.
+          required: true
+          example: gene_list.txt
+        - name: --g2m_genes
+          type: file
+          description: |
+            Path to a .txt file containing the gene list of g2m phase genes to be scored. 
+            The gene list file should be formatted as a single column with gene symbols.
+          required: true
+          example: gene_list.txt
+        - name: --gene_pool
+          type: file
+          description: |
+            File with genes for sampling the reference set. Default is all genes. 
+            The gene pool file should be formatted as a single column with gene symbols.
+          required: false
+          example: gene_pool.txt
+
+    - name: Outputs
+      arguments:
+        - name: --output
+          type: file
+          direction: output
+          description: |
+            Output h5mu file
+          required: true
+          example: output_file.h5mu
+
+    - name: Arguments
+      arguments:
+        - name: --n_bins
+          type: integer
+          default: 25
+          required: false
+          description: |
+            Number of expression level bins for sampling.
+        - name: --random_state
+          type: integer
+          description: |
+            The random seed for sampling.
+          default: 0
+          required: false
+        - name: --use_raw
+          type: boolean
+          description: |
+            Whether to use raw attribute of adata. Defaults to True if .raw is present.
+          required: false
+
+  resources:
+    - path: script.py
+      type: python_script
+  test_resources:
+    - type: python_script
+      path: test.py
+    - path: /resources_test/merge_test_data/
+    
+platforms:
+  - type: docker
+    image: python:3.8
```

Comment:
Could you specify a higher version? So we can wait a while to update it because 3.8 is going to be deprecated the fastest.